### PR TITLE
Support Uniswap v3 in tests

### PR DESCRIPTION
**Pull request by lyubenkov**

### CODE REVIEW

File: scripts/deploy.ts [hunk 1]
```diff
@@ -321,23 +317,29 @@ async function main() {
   tx = await token1.approve(callee.address, MaxUint256)
   await tx.wait(1)
 
-  await pool.connect(LP).collect(beneficiary.address, getMinTick(TICK_SPACINGS[FeeAmount.LOW]), getMaxTick(TICK_SPACINGS[FeeAmount.LOW]), await token0.balanceOf(pool.address),await token1.balanceOf(pool.address))
-  await tx.wait(1)
-  console.log("LP current balances             token0:", toEther(await token0.balanceOf(LP.address)), "   token1:", toEther(await token1.balanceOf(LP.address)))
-  console.log("Beneficiary current balances    token0:", toEther(await token0.balanceOf(beneficiary.address)), "   token1:", toEther(await token1.balanceOf(beneficiary.address)))
+  console.log("Beneficiary collecting rewards");
+  console.log("BEFORE COLLECT")
```
Comment (gigimon) [inline scripts/deploy.ts]: make log more humanize and more understandable
BEFORE COLLECT -> Balances before collect rewards.

File: scripts/deploy.ts [hunk 2]
```diff
@@ -321,23 +317,29 @@ async function main() {
...
+  console.log("Liquidity in pool: ", toEther(await pool.liquidity()))
+  console.log("LP current balances          token0:", toEther(await token0.balanceOf(LP.address)), "   token1:", toEther(await token1.balanceOf(LP.address)))
+  console.log("Beneficiary current balances token0:", toEther(await token0.balanceOf(beneficiary.address)), "   token1:", toEther(await token1.balanceOf(beneficiary.address)))
 
-  console.log("\nBeneficiary removes liquidity from the pool collecting rewards");
+  await pool.connect(LP).burn(getMinTick(TICK_SPACINGS[FeeAmount.LOW]), getMaxTick(TICK_SPACINGS[FeeAmount.LOW]), 0)
+  tx = await pool.connect(LP).collect(beneficiary.address, getMinTick(TICK_SPACINGS[FeeAmount.LOW]), getMaxTick(TICK_SPACINGS[FeeAmount.LOW]), await token0.balanceOf(pool.address),await token1.balanceOf(pool.address))
```
Comment (gigimon) [inline scripts/deploy.ts:336]: Add collect and burn transactions to report

File: scripts/deploy.ts [hunk 3]
```diff
@@ -321,23 +317,29 @@ async function main() {
...
+  console.log("Beneficiary balances token0:", toEther(await token0.balanceOf(beneficiary.address)), "   token1:", toEther(await token1.balanceOf(beneficiary.address)))
 
-  tx = await pool.connect(beneficiary).burn(getMinTick(TICK_SPACINGS[FeeAmount.LOW]), getMaxTick(TICK_SPACINGS[FeeAmount.LOW]), await token0.balanceOf(beneficiary.address))
-  await callee.connect(LP).burn(pool.address, beneficiary.address, await token0.balanceOf(pool.address),await token1.balanceOf(pool.address), MaxUint256, MaxUint256)
-
-  await tx.wait(1)
-  console.log("Beneficiary current balances    token0:", toEther(await token0.balanceOf(beneficiary.address)), "   token1:", toEther(await token1.balanceOf(beneficiary.address)))
-  console.log("Pair total supply:", toEther(await pool.totalSupply()))
+  report["actions"].push({
+    "name": "After collect",
```
Comment (gigimon) [inline scripts/deploy.ts]: After collect in report say nothing

**Review by gigimon — APPROVED** [review]

File: scripts/deploy.ts (hunk 1 above)
Comment (lyubenkov) [reply scripts/deploy.ts]: Fixed, thanks!

File: scripts/deploy.ts (hunk 2 above)
Comment (lyubenkov) [reply scripts/deploy.ts:336]: Added both to the report, thanks!

File: scripts/deploy.ts (hunk 3 above)
Comment (lyubenkov) [reply scripts/deploy.ts]: Renamed to `Collect transaction`

**Review by gigimon — APPROVED** [review]
merge

**Pull request merged**